Makefile

```
@@ -202,7 +209,38 @@ lint-tsc: ## This command executes Typescript linter
 lint-md: ## This command executes Markdown linter
 	$(MARKDOWNLINT_BIN) $(MD_LINT_ARGS)
 
-lint: lint-eslint lint-tsc lint-md ## Runs all linters: ESLint, TypeScript, and Markdown linters in sequence.
+lint-metrics: ## Run rust-code-analysis complexity gate (auto-installs binary if absent)
```

checkmake (0.3.2)

[warning] Target body for "lint-metrics" exceeds allowed length of 5 lines (18).

(maxbodylength)